admin/assets/stylesheets/application.scss

```diff
@@ -258,6 +258,13 @@ input {
   @include button($grey-3);
 
   margin-top: em(20);
+  &:not(:last-child) {
+    margin-right: pem(10px);
+  }
+
+  @include ie(8) {
```

Comment:
@jon-shipley Should this be moved to application-ie8.css?

Reply:
@pris54 - yes that's exactly what happens.  application-ie8.css is a generated file from these includes.